driving_school/data/knowledge_article.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <template id="welcome_article_body">
+      <h1 class="oe-hint" data-oe-version="2.0">Driving School</h1>
+      <p>Hello there!</p>
+      <p>You just installed the Odoo for Driving School package. By doing so, we have installed a bunch of necessary apps to run your bar efficiently.</p>
+      <p>Discover the basics of this package and explore all the possibilities Odoo offers to improve your experience.</p>
+      <div data-oe-role="status" class="o_editor_banner user-select-none lh-1 d-flex align-items-center alert alert-info pb-0 pt-3 o-contenteditable-false">
+          <i data-oe-aria-label="Banner Info" class="o_editor_banner_icon mb-3 fst-normal">💡</i>
+          <div class="w-100 px-3 o_editor_banner_content o-contenteditable-true">
+              <p>If you want to execute a practical guided tour of this module, you should Import demo data and try the Demo - Use Case (In the next Knowledge Article)</p>
+          </div>
+      </div>
+      <div data-embedded="tableOfContent"></div>
+      <p><br /></p>
+      <h1><strong>Basics</strong></h1>
+      <hr />
+      <p>You can access every installed App in your Odoo database on your main dashboard.</p>
+      <ul>
+          <li>Sell lessons, exams and packs from the <strong>Sales App</strong></li>
+          <li>Manage waiver with the <strong>Sign App</strong></li>
+          <li>Let your customers book lessons online thanks to the<strong> Appointment App</strong></li>
+          <li>Manage your employees' shifts with the <strong>Planning App</strong></li>
+          <li>Manage your vehicles with the <strong>Fleet App</strong></li>
+          <li>Generate visibility through the <strong>Website App</strong></li>
+      </ul>
+      <p><br /></p>
+      <h1><strong>Use case</strong></h1>
+      <hr />
+      <p>By uploading this module's demo data, your database has been filled with a few products, sale orders and more.</p>
+      <p>You should be able to execute the following flows to have an overview of various flows you can quickly execute with this package using Odoo.</p>
+      <p>Of course, this is just an overview of the features included in this package. Feel free to add new apps, delete/modify demo data, and test everything around!</p>
+      <div data-oe-role="status" class="o_editor_banner user-select-none lh-1 d-flex align-items-center alert alert-warning pb-0 pt-3 o-contenteditable-false">
+          <i data-oe-aria-label="Banner Warning" class="o_editor_banner_icon mb-3 fst-normal">⚠️</i>
+          <div class="w-100 px-3 o_editor_banner_content o-contenteditable-true">
+              <p>You didn't import demo data? You can still do it. Go to Apps > Industries > Upgrade your Bar &amp; Pub package and check the related box</p>
```

Comment:
Also, here. Industries > Upgrade your **Bar &amp; Pub package** and check the related box